frontend/src/bundles/subscription/components/subscription-plan/subscription-plan.tsx

```diff
@@ -0,0 +1,59 @@
+import { useCallback } from 'react';
+
+import { Button } from '~/bundles/common/components/components.js';
+
+import { type SubscribeRequestDto } from '../../types/types.js';
+
+type Properties = {
+    id: number;
+    name: string;
+    description: string;
+    price: number;
+    stripePriceId: string;
+    handleClick: ({ planId, stripePriceId }: SubscribeRequestDto) => void;
```

Comment:
should be prefixed with on, quality criteria A9
This is likely the first time I've seen this type of type declaration syntax used in code)